utility/src/main/java/org/oppia/android/util/parser/ImageLoader.kt

```diff
@@ -5,13 +5,9 @@ import android.graphics.drawable.Drawable
 import android.graphics.drawable.PictureDrawable
 import androidx.annotation.DrawableRes
 
-/**
- * [ImageTransformation] Represents transformations on an image.
- */
+/** [ImageTransformation] Represents transformations on an image. */
```

Comment:
```suggestion
/** Represents transformations for images loaded using [ImageLoader]. */
```

This might be slightly clearer.

Reply:
Done.